src/components/HURUmap/CategoryHeader/useStyles.js

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography }) => ({
+  root: {},
+  title: {
+    fontSize: typography.pxToRem(28),
+  },
+  image: {
+    position: "relative",
+    height: typography.pxToRem(55.5),
+    width: typography.pxToRem(55.5),
+  },
+  description: {
+    fontWeight: "400",
```

Comment:
Isn't it supposed to be Medium i.e. `500` (which is `body2` in the theme).

Reply:
Fixed